tests/ProgressBar.test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import { render, screen } from "@testing-library/react";
+
+import ProgressBar from "components/ProgressBar";
+
+describe("ProgressBar", () => {
+  it("renders with correct value", () => {
+    render(<ProgressBar progressPercentage={50} progressValue="50%" />);
+
+    expect(screen.getByText("50%")).toBeInTheDocument();
+  });
+});
```

Comment:
Add a test for - `progressValue should be optional. If not provided, it should be equal to progressPercentage%`.